sapi/fpm/fpm/fpm_php.c

```diff
@@ -87,7 +87,9 @@ int fpm_php_apply_defines_ex(struct key_value_s *kv, int mode) /* {{{ */
 
 	if (!strcmp(name, "extension") && *value) {
 		zval zv;
+		zend_interned_strings_switch_storage(0);
```

Comment:
Should this be moved into `php_load_extension`? That seems more future-proof.

Reply:
@iluuu1994 The problem is exactly in FPM. It may load ``PERSISTENT`` extension after php start-up.

Reply:
Yeah I agree with Dmitry - this is FPM specific thing.